README.md

```diff
@@ -0,0 +1,125 @@
+## Rio-tiler
+
+Rasterio plugin to serve mercator tiles from satellite imagery hosted on Amazon S3 buckets.
+
+![circleci][circleci_img]
+![codecov][codecov_img]
+
+[circleci_img]: https://circleci.com/gh/mapbox/rio-tiler.svg?style=svg&circle-token=b78bc1a238c21046a855a9c80b441a8f2f9a4478
+[codecov_img]: https://codecov.io/gh/mapbox/rio-tiler/branch/master/graph/badge.svg?token=zuHupC20cG
+
+Additional support is provided for the following satellite missions: 
+
+* Sentinel 2
+* Landsat 8
+* CBERS
+
+Rio-tiler supports Python 2.7 and 3.3-3.6.
+
+### Install
+
+You can install rio-tiler using pip
+
+```
+    $ pip install -U pip
+    $ pip install rio-tiler
+```
+
+or from source:
+
+```
+
+    $ git clone https://github.com/mapbox/rio-tiler.git
+    $ cd rio-tiler
+    $ pip install -U pip
+    $ pip install -e .
+
+```
+
+here is how to create an AWS Lambda package on most UNIX machines:
+
+```
+    $ pip install rio-tiler --no-binary numpy -t /tmp/vendored -U
+    $ zip -r9q package.zip vendored/*
+```
+
+### Overview
+
+Create tiles using one of these rio_tiler modules: `main`, `sentinel2`, `landsat8`, `cbers`. 
+
+The `main` module can create mercator tiles from any satellite image data source supported by Rasterio (i.e. local files, http, etc.). The mission specific modules make it easier to extract tiles from AWS S3 buckets (i.e. only a scene ID is required); They can also be used to return metadata.
```

Comment:
>  from any satellite image
I think it could me more general, like `any raster sources`  ?

Reply:
Good point